src/EditorFeatures/CSharpTest/EditAndContinue/ActiveStatementTests.cs

```diff
@@ -9930,6 +9930,56 @@ async Task<int> f()
                 Diagnostic(RudeEditKind.UpdatingStateMachineMethodAroundActiveStatement, "f"));
         }
 
+        [Fact]
+        [WorkItem(61415, "https://github.com/dotnet/roslyn/issues/61415")]
+        public void LocalFunction_EditAroundTry_WithActiveStatement()
```

Comment:
>LocalFunction_EditAroundTry_WithActiveStatement [](http://example.com/codeflow?start=20&length=47)

This issue also repros for lambdas, not just local functions, right? Would you mind adding another test to cover that?